include/robot_localization/ros_filter.h

```diff
@@ -492,6 +492,10 @@ template<class T> class RosFilter
     //!
     std::map<std::string, Eigen::MatrixXd> previousMeasurementCovariances_;
 
+    //! @brief We also need the previous covariance matrix for diagnostics
+    //!
+    std::map<std::string, Eigen::MatrixXd> previousCovariances_;
```

Comment:
I could re-use `previousMeasurementCovariances_` if I also update it for when differential mode is off.
Should I re-use it?